StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetConfiguration.swift

```diff
@@ -210,10 +210,6 @@ extension PaymentSheet {
         /// Note: Card brand filtering is not currently supported by Link.
         @_spi(CardBrandFilteringBeta) public var cardBrandAcceptance: PaymentSheet.CardBrandAcceptance = .all
 
-        /// This is an experimental feature that may be removed at any time.
-        /// If true, users can set a payment method as default and sync their default payment method across web and mobile
-        /// If false (default), users cannot set default payment methods.
-        @_spi(AllowsSetAsDefaultPM) public var allowsSetAsDefaultPM = false
```

Comment:
Do a search in the codebase for I see 3 instances that need to be removed/cleaned up. (AllowsSetAsDefaultPM)